spec/models/booth_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+describe 'Booth' do
+  subject { create(:booth) }
+  let!(:conference) { create(:conference) }
+
+  describe 'validation' do
+    it 'has a valid factory' do
+      expect(build(:booth)).to be_valid
+    end
+
+    it { is_expected.to validate_presence_of(:conference) }
+
+    it 'is not valid without a title' do
+      should validate_presence_of(:title)
```

Comment:
Use the is_expected format instead of should, just like you did in your other tests.